pkg/kubelet/kubelet.go

```diff
@@ -110,6 +111,18 @@ const (
 	housekeepingPeriod = time.Second * 2
 
 	etcHostsPath = "/etc/hosts"
+
+	// Capacity of the channel for recieving pod lifecycle events. This number
+	// is a bit arbitrary and may be adjusted in the future.
+	plegChannelCapacity = 1000
+
+	// TODO(yujuhong): migrate some of the parameters to kubelet flags.
+	// Relisting is used to discover missing container events.
+	// Use a shorter period because generic PLEG relies on relisting for
+	// discovering container events.
+	plegRelistPeriod = time.Second * 3
```

Comment:
Why is it of concern? Can you elaborate? Once you add the specialized cache, are there any other issues ?